charts/kubewarden-defaults/values.yaml

```diff
@@ -0,0 +1,43 @@
+# Policy Server settings
+policyServer:
+  name: "default"
+  replicaCount: 1
+  image:
+    repository: ghcr.io/kubewarden/policy-server
+    tag: "v0.2.6"
+  serviceAccountName: policy-server
+  # All permissions are cluster-wide. Even namespaced resources are
+  # granted access in all namespaces at this time.
+  permissions:
+    - apiGroup: ""
+      resources:
+        - namespaces
+        - services
+    - apiGroup: "networking.k8s.io"
+      resources:
+        - ingresses
+  telemetry:
+    enabled: True
+  env:
+    - name: KUBEWARDEN_LOG_LEVEL
+      value: info
+  annotations: {}
+
+recommendedPolicies:
+  enabled: False
+  skipNamespaces:
+    - kube-system
+    - rancher
```

Comment:
Last detail, I can identify: `kube-system`, `kube-public`, `kube-node-lease`. I am not sure about what are the ones created by rancher, is it only `rancher`? I remember seeing `cattle-system` or `rancher-cattle-system`, but I'm not sure what's the current status as of today of created namespaces.

Reply:
I do not know, I'm looking into this now. I just update the PR to check with you all agree with the approach. I put these values just as examples. I should made that more clear. Sorry.

Reply:
https://github.com/rancher/rancher/blob/e9dfe98420468f3bcc031f9fa332bd8be7cd962d/pkg/settings/setting.go#L79

Reply:
Thank for @cmurphy !